tests_consume/test_via_rlp.py

```diff
@@ -0,0 +1,244 @@
+"""
+Test module that defines a test to execute a fixture against an EVM blocktest-like command.
+"""
+import json
+import tempfile
+from pathlib import Path
+from typing import List, Literal, Union
+
+import pytest
+import requests
+from hive.client import Client
+from hive.testing import HiveTest, HiveTestResult, HiveTestSuite
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+from ethereum_test_tools.common.types import Fixture
+from pytest_plugins.consume_via_rlp.consume_via_rlp import TestCase
+from pytest_plugins.consume_via_rlp.network_ruleset_hive import ruleset
+
+
+@pytest.fixture(scope="function")
+def temp_dir() -> tempfile.TemporaryDirectory:
```

Comment:
I think we could skip the temp dir by putting the file's contents in a `bytes`, I think hive.py should support this.